### PR TITLE
Documentation for TwitchLib

### LINKED ISSUE #22: Add examples for usage with TwitchLib
**Describe the solution you'd like**
I'd like examples for [TwitchLib](/TwitchLib/TwitchLib) on https://fdgt.dev.

### PR DESCRIPTION
**Pull request by itssimple**
This pull request contains documents with examples of how to connect to [fdgt](https://fdgt.dev) with [TwitchLib](https://github.com/TwitchLib/TwitchLib) and [TwitchLib.Communication.Clients.FDGTClient](https://github.com/itssimple/TwitchLib.Communication.Clients.FDGTClient)

Fixes #22

### CODE REVIEW

File: docs/code-templates/TwitchLib/FDGTConnect.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+```csharp
+// Additional package to download
+// - TwitchLib.Communication.Clients.FDGTClient
```
Comment (trezy) [inline docs/code-templates/TwitchLib/FDGTConnect.md:4]: Will this be enough for users to understand what to do? I'm not familiar with the C# ecosystem, so I have no idea. 😬

Comment (itssimple) [reply docs/code-templates/TwitchLib/FDGTConnect.md:4]: I guess I could add a little more text to it. ^^

Comment (trezy) [reply docs/code-templates/TwitchLib/FDGTConnect.md:4]: Don't feel obligated, my question is really this: If a C# user comes to the `fdgt` documentation and reads this comment, will they know where to download it from this additional package? Will they know how to download it (package manager, etc)? Is there additional configuration needed?

If there is general C# knowledge that will allow a typical user to understand this comment, then I'm good with it. If not, let's bulk it out to ensure that the typical C# user will understand. 😁

Comment (itssimple) [reply docs/code-templates/TwitchLib/FDGTConnect.md:4]: Yeah, most people know how to download NuGet-packages at least. It's like yarn/npm. ^^

**Pull request merged**